core/src/snapshot_packager_service.rs

```diff
@@ -157,31 +153,6 @@ impl SnapshotPackagerService {
         Error::IO(IOError::new(ErrorKind::Other, error))
     }
 
-    fn serialize_status_cache(
```

Comment:
I moved this method to  ledger/src/snapshot_utils.rs to collect any serialize/deserialize code there.